lib/alexaSmartHomeV2.js

```diff
@@ -276,20 +278,27 @@ function AlexaSH2(adapter) {
                 }
             } else {
                 if (type === 'number') {
-                    if (states[id].common.unit === 'C' || states[id].common.unit === 'C°' || states[id].common.unit === '°C' ||
-                        states[id].common.unit === 'F' || states[id].common.unit === 'F°' || states[id].common.unit === '°F' ||
-                        states[id].common.unit === 'K' || states[id].common.unit === 'K°' || states[id].common.unit === '°K') {
+                    if (/^level\.color\./.test(role)) {
+                        // test for color first, since color temperature may be given in Kelvin (K)
+                        if (role === 'level.color.hue' || role === 'level.color.rgb') {
+                            actions = ['setColor', /*'incrementColorTemperature', 'decrementColorTemperature', 'setColorTemperature', */'turnOn', 'turnOff'];
+                        } else if (role === 'level.color.temperature') {
+                            actions = ['setColorTemperature', /*'incrementColorTemperature', 'decrementColorTemperature'*/];

```

Comment:
Where are those actions defined? Does the server code also have to be updated to support those?

Reply:
You're working with the v2 skill api (which is depreacted and you actually can't create a skill with that version anymore) not the controller you're looking at (v3). The Actions are defined in the v2 Skill Documentation which in return is very hard to find right now...

These actions are defined here:
https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/archive/shv2-smart-home-skill-api-reference#discovery-messages
Scroll down to: discoveredAppliance.actions

Reply:
Thanks a lot!